modules/olm-fb-catalogs.adoc

```diff
@@ -7,11 +7,6 @@
 
 _File-based catalogs_ are the latest iteration of the catalog format in Operator Lifecycle Manager (OLM). It is a plain text-based (JSON or YAML) and declarative config evolution of the earlier SQLite database format, and it is fully backwards compatible. The goal of this format is to enable Operator catalog editing, composability, and extensibility.
 
-[NOTE]
-====
-The default Red Hat-provided Operator catalogs for {product-title} 4.6 and later are currently still shipped in the SQLite database format.
-====
-
```

Comment:
If someone is reading the <=4.10 docs, they might get the impression that 4.11 is still using SQLite because this text says "4.6 and later"

Not sure it matters _that_ much, but is there a process to update previous version docs such that we could amend that statement to `4.6 through 4.10 are shipped in the SQLite database format.`

Reply:
@joelanford Good point. I will add a story to clean up earlier versions of the documentation.

Reply:
@michaelryanpeter did we update the release notes to say from 4.11 all indexes are in FBC format?

Reply:
@camilamacedo86 Yes. @sheriff-rh made a PR for that which has merged: https://github.com/openshift/openshift-docs/pull/48040